tasks/main.yml

```diff
@@ -65,9 +72,57 @@
     - bigbluebutton
     state: present
 
+# nojournal not supported for default engine
+# inMemory engine seems only in Enterprise distribution
+- debug: var=mongodb_version
+- name: Ensure mongodb4+ does not use nojournal - systemd
+  replace:
+    dest: /etc/systemd/system/mongod.service
+    regexp: ' --nojournal$'
+    replace: ''
+    backup: yes
+  when: mongodb_version is defined and mongodb_version|string == '4.2'
+  notify:
+    - reload systemd
+    - restart mongo
+
+- name: Ensure mongodb4+ does not use nojournal - config
+  template:
+    src: mongo4-ramdisk.conf.j2
+    dest: /usr/share/meteor/bundle/mongo-ramdisk.conf
+    backup: yes
+    mode: '0644'
+  notify:
+    - restart mongo
+  when: mongodb_version is defined and mongodb_version|string == '4.2'
+
+- name: LXD | disable freeswitch scheduling options
+  replace:
+    dest: /lib/systemd/system/freeswitch.service
+    regexp: "{{ item.re }}"
+    replace: "{{ item.rep }}"
+    backup: yes
+    mode: '0644'
+  with_items:
+    - { re: '^IOSchedulingClass=realtime', rep: '#IOSchedulingClass=realtime' }
+    - { re: '^CPUSchedulingPolicy=rr', rep: '#CPUSchedulingPolicy=rr' }
+  notify:
+    - reload systemd
+
+- meta: flush_handlers
+
+- name: Ensure services are enabled
+  service:
+    name: mongod
+    enabled: true
+    state: started
+
 - import_tasks: letsencrypt.yml
   when: bbb_letsencrypt_enable
 
+- import_tasks: certificate-selfsigned.yml
+  when: not bbb_letsencrypt_enable
```

Comment:
Please make this step configurable, so that users can use pregenerated certificates with `bbb_ssl_cert` and `bbb_ssl_key`...

Reply:
@juju4 Could you take a look at this? Then I can merge your PR...